haystack/pipeline.py

```diff
@@ -593,6 +598,43 @@ def run(self, **kwargs):
         return kwargs, "output_1"
 
 
+class QueryClassifier(BaseComponent):
+    """
+    A node to choose between two nodes based on query classification result.
+    """
+
+    outgoing_edges = 2
+
+    def __init__(
+        self, 
+        query_classifier: Optional[GradientBoostingClassifier] = pickle.load(
+            urllib.request.urlopen(
+                "https://ext-models-haystack.s3.eu-central-1.amazonaws.com/gradboost_query_classifier/model.pickle"
+            )
+        ), 
+        query_vectorizer: Optional[TfidfVectorizer] = pickle.load(
+            urllib.request.urlopen(
+                "https://ext-models-haystack.s3.eu-central-1.amazonaws.com/gradboost_query_classifier/vectorizer.pickle"
+            )
+        )
+    ):
+        """
+        :param query_classifier: Gradient boosting based binary classifier to classify between question vs statement queries.
+        :param query_vectorizer: A ngram based Tfidf vectorizer for extracting features from query.
+        """
+        self.query_classifier = query_classifier
+        self.query_vectorizer = query_vectorizer
```

Comment:
These lines can be replace as follows -
```
self.query_classifier = query_classifier or pickle.load(
            urllib.request.urlopen(
                "https://ext-models-haystack.s3.eu-central-1.amazonaws.com/gradboost_query_classifier/model.pickle"            
           )
)

self. query_vectorizer = query_vectorizer or pickle.load(
            urllib.request.urlopen(
                "https://ext-models-haystack.s3.eu-central-1.amazonaws.com/gradboost_query_classifier/vectorizer.pickle"            
           )
)
```

Reply:
done, thanks for all your guidelines and feedback.